data/json/mutations.json

```diff
@@ -662,7 +662,7 @@
         "type" : "mutation",
         "id" : "LACTOSE",
         "name" : "Lactose Intolerance",
-        "points" : -1,
+        "points" : -2,
```

Comment:
Add small amounts of it to veggies and wheat and quite a bit to bone comestibles (bone broth, aspic).

Even if milk was the only source of calcium, lactose intolerance wouldn't be worth -2 points. Most lactose items rot within the first season, meaning you'd pretty much be guaranteed deficiency anyway. Making it a good source of calcium makes it OK for it to have that -1 cost rather than being free points.

Plus, requiring lactose tolerance to avoid deficiency is just plain unrealistic.